artifact/gem5art/artifact/artifact.py

```diff
@@ -229,8 +229,12 @@ def __repr__(self) -> str:
         return vars(self).__repr__()
 
     def _getSerializable(self) -> Dict[str, Union[str, UUID]]:
-        data = vars(self)
+        data = vars(self).copy()
         data['inputs'] = [input._id for input in self.inputs]
+        if isinstance(data['cwd'], Path):
```

Comment:
These `isinstance`s shouldn't be here. We *know* it's a `Path`, right?

Reply:
Yeah, this is redundant. Will update this